conf/application.conf

```diff
@@ -188,3 +188,9 @@ microservice {
   }
```

Comment:
you can stub the get eligibility endpoint using wiremock and return `MissingUserInfos` as a response . even if you as above we dont know how frontend reacts to mising info response, right ?